source/applicability/basic_structure.rst

```diff
@@ -1187,13 +1187,16 @@ Example of message exchange with subscription, status updates and unsubscription
 Command messages
 ^^^^^^^^^^^^^^^^
 
-Command messages are used to give order using one or more commands, for the
-referenced object.
+Command messages are used to give order to the referenced object.
 The site responds with a command acknowledgement.
 
 All arguments needs to included in a command, otherwise it results a serious
 error resulting in MessageNotAck. See section about :ref:`incomplete-commands`.
 
+Only a single command is allowed in single CommandRequest, otherwise any
```

Comment:
=> Only a single command is allowed in a CommandRequest, otherwise any

Reply:
Fixed